docs/RELEASE.md

```diff
@@ -1,28 +1,40 @@
 # Release Process
 
-## QA
-1. Decide what commit should be used for the release and create a draft release
-   - This commit is normally a `master` branch commit, but there are exceptions (e.g. hotfixes)
-   - The release description MUST contain a list of new features
-2. Share release and a Crashlytics Beta link to that commit with the QA team
-   - New features and core features (create Safe, send transactions, restore Safe) MUST be tested with the Mainnet version
-   - Other features SHOULD be tested with the Rinkeby version
-3. If required fix bugs and repeat process
+## Release candidate
+1. Create a release task in GitHub using the "New Release" template
+2. Create and push the `release` branch
+3. Update APP_VERSION_NAME (x.y.z)
+4. Build pipeline will produce and upload builds with `rc` prefix (x.y.z-buildNumber`rc`-buildVariant)
+5. Notify QA about release candidate
+6. Create pull-request for release branch in case additional work/fixes are required for release candidate and go to step 4.
+7. Normal development continues targeting `master` branch.
+
+## Publishing release
+1. QA approved release candidate
+2. Get release notes from Product Owner
+3. Draft new Release in GithHub
+  - Select release branch as target
+  - Tag release using `vX.Y.Z` format
+  - Add title using `Version X.Y.Z` format
+  - Add release notes from Product Owner to description 
+4. Team will be notified in slack about new release
+```
+Version X.Y.Z for gnosis/safe-android is out!
+```
+5. Download mainnet apk (prefixed with `-release`) from release page (https://github.com/gnosis/safe-android/releases/tag/vX.Y.Z) and upload it to playstore
+  - Check if version should be published as Alpha/Beta to signed up users first
```

Comment:
Can we add here to add external testers in firebase console for mainnet and rinkeby?